src/System.Runtime/tests/System/Reflection/SignatureTypes.netcoreapp.cs

```diff
@@ -0,0 +1,529 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Xunit;
+
+namespace System.Reflection.Tests
+{
+    public static class SignatureTypeTests
+    {
+        [Fact]
+        public static void IsSignatureType()
```

Comment:
@AtsushiKan Might consider using a `Theory` to loop through the data.

Reply:
See comment just below.